website/src/pages/dictionary/index.js

```diff
@@ -64,10 +63,16 @@ export const ModalPortal = ({ children }) => {
 };
 
 const data = require('./data.json');
+const dictionaryTreeData = require('./tree.json');
 
 async function fetchDictionary(version) {
-  const response = await axios.get(`/data/schemas/${version}.json`);
-  return response.data;
+  try {
+    const dict = await axios.get(`/data/schemas/${version}.json`);
+    const tree = await axios.get(`/data/schemas/${version}_tree.json`);
+    return { dict: dict.data, tree: tree.data };
+  } catch (e) {
+    throw e;
```

Comment:
if just gonna throw, there's no need to catch :P

Reply:
@hlminh2000 need an empty `finally` if not doing this. this is more explicit. I could have sworn you could just do a `try` though